.github/workflows/codeql.yml

```diff
@@ -0,0 +1,72 @@
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
+name: "Weekly CodeQL"
+
+on:
+  schedule:
+    # Weekly on Sunday.
+    - cron: '0 10 * * 0'
```

Comment:
On second thought, maybe nightly would be better? This workflow is not too large (does not take too many resources). Having nightly builds will help bisection in case there is some breakage.

Reply:
Done. 

The workflow runs every day at 1am

Reply:
Done. 

The workflow runs every day at 1am